docs/simulators/DualSPHysics.md

```diff
@@ -48,6 +52,395 @@ task.download_outputs()
+read -n1 -r -p "Press any key to continue..." key
+```
+
+As you can see above, the main simulation steps involve these two commands:
+```bash
+# CODES are executed according the selected parameters of execution in this testcase
+${gencase} ${name}_Def ${dirout}/${name} -save:all
+if [ $? -ne 0 ] ; then fail; fi
+
+${dualsphysicscpu} ${dirout}/${name} ${dirout} -dirdataout data -svres
+if [ $? -ne 0 ] ; then fail; fi
+```
+
+The post-processing commands follow, with calls to `partvtk`, `partvtkout`,
+`boundaryvtk` and `isosurface`. These are the commands that generate the data
+required for producing the visualization shown above with ParaView.
+
+There are also a number of variables being set in the beginning of the script. 
+Some of them have to do with the path to DualSPHysics commands, and they
+are assuming a local instalation. For our Inductiva python script, we won't 
+need any of that, since the API takes care of all command paths for us in the
+remote machine. There are also a number of variable related to filenames and 
+input/output directories:
+```bash
+export name=CaseTurbine
+export dirout=${name}_out
+export diroutdata=${dirout}/data
+```
+For a matter of convenience, we will want to keep the control over input / 
+output directories, so we will convert and keep these variables in our python
+script. We will also take the chance to explicitly add a few more output 
+directories, and we will make everything a bit more pythonic. So, in our python script, we should have something like this: 
+```python
+# Let's keep the original variables for the directory names, but we will
+# make them more pythonic and we will add a few others for readibility
+name="CaseTurbine"
+dirout=f"{name}_out"
+data_dirout=f"{dirout}/data"
+particles_dirout=f"{dirout}/particles"
+boundary_dirout=f"{dirout}/boundary"
+surface_dirout=f"{dirout}/surface"
+```
+
+Now, we need to convert the 7 command lines to strings, while making use of
+the variables we have just defined. Also, will be able to we call the 
+DualSPHysics commands (gencase, dualsphysics, partvtk, etc) directly, since they
+are all pre-installed the the machine we will spin up later.
+
+```python
+commands = [
+    f"gencase {name}_Def {dirout}/{name} -save:all",
+    f"dualsphysics {dirout}/{name} {dirout} -dirdataout data -svres",
+    f"partvtk -dirin {data_dirout} -savevtk {particles_dirout}/PartFloating -onlytype:-all,+floating",
+    f"partvtk -dirin {data_dirout} -savevtk {particles_dirout}/PartFluid -onlytype:-all,+fluid",
+    f"partvtkout -dirin {data_dirout} -savevtk {particles_dirout}/PartFluidOut -SaveResume {particles_dirout}/_ResumeFluidOut",
+    f"boundaryvtk -loadvtk AutoActual -motiondata {data_dirout} -savevtkdata {boundary_dirout}/Turbine -onlytype:floating -savevtkdata {boundary_dirout}/Bound.vtk -onlytype:fixed",
+    f"isosurface -dirin {data_dirout} -saveiso {surface_dirout}/Surface"
+]
+
+```
+The rest of the python script follows the usual patter. Here is the finaly 
```

Comment:
The rest of the Python script follows the usual pattern. Here is the final resulting script, ready to be executed:

Reply:
Done!